.ci/magic-modules/generate-terraform.sh

```diff
@@ -62,6 +60,15 @@ git checkout -B "$(cat ../../branchname)"
 
 apply_patches "$PATCH_DIR/terraform-providers/$PROVIDER_NAME" "$TERRAFORM_COMMIT_MSG" "$LAST_COMMIT_AUTHOR" "master"
 
+# We need to vendor after patching in case two simultaneous PRs cause a change in vendored directories, especially
+# removals.
+# This way, the Magician will generate the vendor directory based on the end state of the repo after the PR is merged
+# instead of based on the tentative output of MM.
+GO111MODULE=on go mod vendor
+git add -A
+# Set the "author" to the commit's real author.
+git commit -m "Update vendored dependencies" --author="$LAST_COMMIT_AUTHOR" || true  # don't crash if no changes
```

Comment:
Sorry - needs to be a single commit so that create-prs works right, so there's some additional complexity.  You can do
`git add -A`, `git commit --amend --no-edit  --author="$LAST_COMMIT_AUTHOR" || true`, I _think_...

Reply:
Done!